models/maat/R/model2netcdf.MAAT.R

```diff
@@ -62,7 +67,7 @@ model2netcdf.MAAT <- function(rundir, outdir, sitelat = -999, sitelon = -999, st
       PEcAn.logger::logger.info(paste0("Skipping conversion for: ", newname))
       dat.new <- dat
     } else {
-      dat.new <- try(PEcAn.utils::misc.convert(dat,oldunits,newunits),silent = TRUE)
+      dat.new <- apply(as.matrix(dat,length(dat),1),1,f_sort)
```

Comment:
Is `misc.convert` already vectorized, and robust to infinite/NA values? If so, why not just do this?

```r
dat.new <- PEcAn.utils::misc.convert(dat, oldunits, newunits)
dat.new[!is.finite(dat.new)] <- -999
```

Then, you also don't need the `f_sort` function above.